data/presets/shop.json

```
@@ -38,6 +38,6 @@
         "retailer",
         "store"
     ],
-    "name": "Shop",
+    "name": "{shop/_yes}",
```

```suggestion
    "name": "{shop/yes}",
```

When referencing a name, you have to omit the underscore. This is explained in https://github.com/ideditor/schema-builder/pull/148/files but not merged yet.